tests/conftest.py

```diff
@@ -67,6 +75,82 @@ def fixture_login_page(selenium, server_url, org):
     return login_page.open()
 
 
+@pytest.fixture(name='create_user', scope='session')
+def fixture_create_user(root_session, server_url):
+    """Return a function to create a user."""
+
+    def create_user(name=None, email=None, password=None):
+        """Create a user via api.
+
+        This will use the authenticated root user requests session to create a
+        user, accept the invite, and add a password.
+    
+        Args:
+            server_url: 
+                URL for redash instance
+            session: 
+                Requests login session. 
+                This is needed to allow for user creation.
+            user:
+                User object that will be created.
+
+        """
+        data = {
+            "name": name,
+            "email": email,
+            "no_invite": True,
+        }
+        # create user
+        response = root_session.post(
+            f'{server_url}/api/users',
+            json=data)
+        if response.status_code != 200:
+            print(f"unable to create user: {response.text}")
```

Comment:
We need to make sure the test suite exits with an error if we user creation fails.

Please change this to raise an ``Exception``.